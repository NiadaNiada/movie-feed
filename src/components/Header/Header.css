.header {
    width: 100%;
    height: auto;
    background: black;
    padding: 0 20px;
    box-sizing: border-box;
}

.header-content {
    max-width: 1280px;
    min-height: 120px;
    height: auto;
    padding: 20px 0px;
    margin: 0 auto;
    box-sizing: border-box;
    overflow: hidden;
    display: flex;
}

.header-content a {
    text-decoration: none;
}

.header-logo {
    width: 300px;
    height: 100px;
    margin-top: 20px;
    float: left;
    background-image: url("../../images/tmdbIcon.png");
    background-size: contain;
    background-repeat: no-repeat;
}

.header-title {
    color: white;
    font-size: x-large;
    text-decoration: none;
}
@media screen and (max-width: 500px) {
    .header-content {
        max-width: 1280px;
        min-height: 0px;
    }

    .logo {
        display: inline-block;
        width:80px;
        margin-top: 0px;
    }

    .logo {
        width: 150px;
        margin-top: 5px;
    }
}