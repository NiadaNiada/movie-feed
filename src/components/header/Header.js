import React from 'react';
import { Link } from "react-router-dom";
import "./Header.css";

const Header = () => {
    return (
        <div className="rmdb-header">
            <div className="rmdb-header-content">
                <Link to="/">
                    <div
                        className="rmdb-logo"
                    />
                    <div className="title">
                    Enjoy best movies
                    </div>
                </Link>
                <div
                    className="rmdb-tmdb-logo"
                />
            </div>
        </div>
    );
}

export default Header;