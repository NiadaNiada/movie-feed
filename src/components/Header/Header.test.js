import React from "react";
import Header from "./Header";
import {render} from "@testing-library/react";
import {MemoryRouter} from "react-router";

describe('<Header />', function () {
    test('renders Header page', () => {
        const {firstChild} = render(<MemoryRouter><Header/></MemoryRouter>);
        expect(firstChild).toMatchSnapshot();
    });

    test('renders Header page and check fot the valid title', () => {
        const {getByText} = render(<MemoryRouter><Header/></MemoryRouter>);
        expect(getByText('Enjoy best movies')).toBeTruthy();
    });
});