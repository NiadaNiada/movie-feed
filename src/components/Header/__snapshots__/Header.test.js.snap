// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders Header page 1`] = `undefined`;
